frontend/plan/components/schedule/ScheduleDisplay.tsx

```diff
@@ -316,6 +322,11 @@ const ScheduleDisplay = ({
                     friendshipState.activeFriendSchedule.found && (
                         <FriendEmptySchedule message="Your friend has not added courses to their schedule yet" />
                     )}
+                {!notEmpty &&
+                    readOnly &&
+                    friendshipState.activeFriendSchedule.found && (
```

Comment:
What is this case for? Should it be `!notEmpty && isPathSchedule`

Reply:
Yeah exactly - was in the middle of changing (prob shouldn't have committed yet).